doc/neomake.txt

```diff
@@ -56,7 +56,7 @@ fantastic vim plugins such as syntastic and dispatch.
 <
                                                              *:NeomakeCancelJob*
 :NeomakeCancelJob *{job_id}*
```

Comment:
[You bet!](https://github.com/machakann/vim-vimhelplint) :)

Will add the following to the Makefile shortly:
```make
build:
	mkdir $@
build/vim-vimhelplint: build
	git clone https://github.com/machakann/vim-vimhelplint $@
vimhelplint: build/vim-vimhelplint
	vim -esN --cmd "set rtp+=build/vim-vimhelplint" -c 'filetype plugin on' \
		-c 'e doc/neomake.txt' -c 'verb VimhelpLintEcho' -c q
```